src/java.desktop/windows/native/libawt/java2d/d3d/D3DShaderGen.c

```diff
@@ -56,9 +56,8 @@ static char *strHeaderFile = "D3DShaders.h";
 
 // REMIND
 //#define J2dTraceLn(a, b) fprintf(stderr, "%s\n", b);
-//#define J2dTraceLn1(a, b, c) fprintf(stderr, b, c);
-#define J2dTraceLn(a, b)
-#define J2dTraceLn1(a, b, c)
+//#define J2dTraceLn(a, b, c) fprintf(stderr, b, c);
```

Comment:
It looks like the above commented J2dTraceLn for arguments 0 and 1 now conflicts?

Reply:
Should be better now